Visual Studio 2010: Crashes in AllTests Release executable

There are quite a few crashing test cases in UTestTest.cpp and some more in MemoryLeakWarningTest.cpp. There may be more. I was alerted to the crashes by a co-worker who uses CppUTest with VS2010.

I am currently investigating the issue.

I am also experiencing a strange linker problem in the Debug configuration:

```
2>LINK : fatal error LNK1123: failure during conversion to COFF: file invalid or corrupt
```

--- a/CppUTest.vcxproj
+++ b/CppUTest.vcxproj
@@ -89,7 +89,7 @@ copy $(OutDir)vc100.pdb lib\vs2010
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <ClCompile>
-      <Optimization>MaxSpeed</Optimization>
+      <Optimization>Disabled</Optimization>
       <AdditionalIncludeDirectories>.\include;.\include\Platforms\VisualCpp;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <InlineFunctionExpansion>OnlyExplicitInline</InlineFunctionExpansion>
       <PreprocessorDefinitions>WIN32;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>

